src/convertToObject.js

```diff
@@ -7,6 +7,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stylesLines = sourceString.split(';').map((item) => item.trim());
+
+  const convertedStylesToObject = {};
+
+  stylesLines.forEach((line) => {
```

Comment:
Using a `forEach` loop is acceptable, but you could consider using more declarative methods like `reduce` to build the `convertedStylesToObject` directly from `stylesLines`, which aligns with the checklist requirement to use iteration methods instead of loops.

Reply:
> Using a `forEach` loop is acceptable, but you could consider using more declarative methods like `reduce` to build the `convertedStylesToObject` directly from `stylesLines`, which aligns with the checklist requirement to use iteration methods instead of loops.

Good point, you can use `reduce` to generate an object